### PR TITLE
Eronous is bored

### PR DESCRIPTION
**Pull request by Eronous**
Its back, with fewer spelling mistakes

### CODE REVIEW

File: src/main/resources/systems/er11.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
         "asteroidbeta",
         "asteroidbwh"
     ],
-    "imagePath": "ER11_betaasteroid.png",
+    "imagePath": "er)11_async.png",
```
Comment (jamesrudolph) [inline src/main/resources/systems/er11.json:8]: this looks wrong

File: src/main/resources/planets/rialarhon.json
Comment (jamesrudolph) [inline src/main/resources/planets/rialarhon.json:1]: changing the ID will break all existing "rialarhon" but maybe that's fine?

File: src/main/resources/systems/er68.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
     "id": "er68",
-    "name": "Nebula2e",
+    "name": "Supernova2e",
```
Comment (jamesrudolph) [inline src/main/resources/systems/er68.json:3]: what's the "2e" all about?

Comment (Eronous) [reply src/main/resources/systems/er68.json:3]: 2e just means second [tile] eronous

File: src/main/resources/planets/rialarhon.json
Comment (Eronous) [reply src/main/resources/planets/rialarhon.json:1]: I dont know what it will do

File: src/main/resources/systems/er11.json (hunk 1 above)
Comment (Eronous) [reply src/main/resources/systems/er11.json:8]: er_11_async.png

**Pull request merged**